app/src/main/java/in/testpress/testpress/ui/OrderConfirmActivity.java

```diff
@@ -0,0 +1,293 @@
+    @InjectView(id.zip) EditText zip;
+    @InjectView(id.landmark) EditText landmark;
+    @InjectView(id.phone) EditText phone;
+    @InjectView(id.fill_all_details) TextView fillAllDetailsText;
+    @InjectView(id.continueButton) Button continueButton;
+    @InjectView(id.pb_loading) ProgressBar progressBar;
+    @InjectView(id.shippingDetails) RelativeLayout shippingDetails;
+    final TextWatcher watcher = validationTextWatcher();
+    PaymentParams paymentParams;
+    PayuConfig payuConfig;
+    ProductDetails productDetails;
+    Order order;
+    OrderItem orderItem = new OrderItem();
+
+    @Override
+    public void onCreate(Bundle bundle) {
+        super.onCreate(bundle);
+        Injector.inject(this);
+        setContentView(R.layout.shipping_address);
+        ButterKnife.inject(this);
+        progressBar.setVisibility(View.VISIBLE);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+        shippingDetails.setVisibility(View.GONE);
+        progressBar.getIndeterminateDrawable().setColorFilter(getResources().getColor(R.color.primary), PorterDuff.Mode.SRC_IN);
+        productDetails = getIntent().getParcelableExtra("productDetails");
+        orderItem.setProduct(Constants.Http.URL_BASE + "/api/v2/products/" + productDetails.getSlug() + "/"); //now using v2 instead v2.1
+        orderItem.setQuantity(1);
+        orderItem.setPrice(productDetails.getPrice());
+        final List<OrderItem> orderItems = new ArrayList<>();
+        orderItems.add(orderItem);
+        new SafeAsyncTask<Order>() {
+
+            @Override
+            public Order call() throws Exception {
+                AccountManager manager = AccountManager.get(OrderConfirmActivity.this);
+                Account[] accounts = manager.getAccountsByType(Constants.Auth.TESTPRESS_ACCOUNT_TYPE);
+                if(accounts.length != 0) {
+                    return serviceProvider.getService(OrderConfirmActivity.this).order(Constants.Http.URL_USERS + accounts[0].name + "/", orderItems);
+                } else {
+                    serviceProvider.handleForbidden(OrderConfirmActivity.this, serviceProvider, logoutService);
+                    throw new Exception("No Account exist");
+                }
+            }
+
+            @Override
+            protected void onSuccess(Order createdOrder) {
+                order = createdOrder;
+                if(createdOrder.getStatus().equals("Completed")) {
+                    setResult(RESULT_OK);
+                    Intent intent;
+                    if(productDetails.getExams().size() != 0) {
+                        if(productDetails.getExams().size() > 1) {
+                            intent = new Intent(OrderConfirmActivity.this, ExamsListActivity.class);
+                        } else {
+                            intent = new Intent(OrderConfirmActivity.this, ExamActivity.class);
+                            intent.putExtra("exam", productDetails.getExams().get(0));
```

Comment:
This case is not advisable. There may be products where the exams may start in the future. And this will fail. It is better to show the order completed details and let them go to the exams list page from there by clicking a button there.